cms/djangoapps/maintenance/views.py

```diff
@@ -0,0 +1,126 @@
+"""
+Index view for the maintenance app.
+"""
+
+from student.roles import GlobalStaff
+
+from edxmako.shortcuts import render_to_response
+
+from django.core.urlresolvers import reverse_lazy
+from django.utils.translation import ugettext_lazy as _
+
+from django.contrib.auth.decorators import login_required
+from django.core.exceptions import PermissionDenied
+
+
+MAINTENANCE_INDEX_URLS = [
+    {
+        "url": reverse_lazy("maintenance:show_orphans"),
+        "name": _("Print Orphans"),
+        "description": _("View orphans."),
+    },
+    {
+        "url": reverse_lazy("maintenance:delete_orphans"),
+        "name": _("Delete Orphans"),
+        "description": _("Delete orphans."),
+    },
+    {
+        "url": reverse_lazy("maintenance:export_course"),
+        "name": _("Export Course"),
+        "description": _("Export course"),
+    },
+    {
+        "url": reverse_lazy("maintenance:import_course"),
+        "name": _("Import Course"),
+        "description": _("Import Course."),
+    },
+    {
+        "url": reverse_lazy("maintenance:delete_course"),
+        "name": _("Delete Course"),
+        "description": _("Delete course."),
+    },
+    {
+        "url": reverse_lazy("maintenance:force_publish_course"),
+        "name": _("Force Publish Course"),
+        "description": _("Force publish course."),
+    },
+]
+
+
+@login_required
+def index(request):
+    """Render the maintenance index view. """
+    # Only global staff (PMs) are able to activate/deactivate certificate configuration
+    if not GlobalStaff().has_user(request.user):
+        raise PermissionDenied()
+    context = {
+        "urls": MAINTENANCE_INDEX_URLS,
+    }
+    return render_to_response("maintenance/index.html", context)
+
+@login_required
+def show_orphans(request):
+    """Render show orphans view """
+    # Only global staff (PMs) are able to activate/deactivate certificate configuration
+    if not GlobalStaff().has_user(request.user):
```

Comment:
Thanks for this decorator. I was  going to find this after I had finished the skeleton of the app. But you saved my time :)